<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>255</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Вычисение массы элемента</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(5, 49, 95);</string>
  </property>
  <widget class="QLineEdit" name="edit_f">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>170</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(101, 150, 255);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="f_txt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Введите формулу соединения:</string>
   </property>
  </widget>
  <widget class="QLabel" name="s_txt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Введите массу соединения:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_s">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>170</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(101, 150, 255);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(101, 150, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>г</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>кг</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(101, 150, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Получить ответ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="t_txt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Массу какого соединения требуется узнать:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_t">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>170</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(101, 150, 255);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_result">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
